cabal-install/src/Distribution/Client/ProjectPlanning.hs

```diff
@@ -4283,7 +4283,8 @@ setupHsBuildFlags par_strat elab _ verbosity builddir =
     , buildDistPref = toFlag builddir
     , buildNumJobs = mempty -- TODO: [nice to have] sometimes want to use toFlag (Just numBuildJobs),
     , buildUseSemaphore =
-        if elabSetupScriptCliVersion elab >= mkVersion [3, 9, 0, 0]
+        if elabSetupScriptCliVersion elab >= mkVersion [3, 11, 0, 0]
+          -- Cabal 3.11 is the first version that supports parallelism semaphores
           then par_strat
```

Comment:
```suggestion
          then -- Cabal 3.11 is the first version that supports parallelism semaphores
            par_strat
```

Reply:
CI fails because of formoulu, this should fix it.

Reply:
@mpickering care to set the "merge_me` label, after committing this change, if it looks right to you?